### PR TITLE
[Docs] Add UI screenshot to UI Contributor docs

### LINKED ISSUE #1448: [Docs] Add UI screenshot to UI Contributor docs
**Current State:**
The [UI section of the CONTRIBUTING.md](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#ui-contribution-flow) is somewhat uninspiring. 

**Desired State:**
1. The contributing to UI section could use a screenshot of the Meshery UI
1. In addition to a screenshot in this section, the README should list Billboardjs as on one of the frontend technologies in use.

**Implementation**
1.  A screenshot of Meshery may be retrieved from https://meshery.io. Place a copy of the desired screenshot into the /ui/assets/img/readme directory in this repository.
1. List Billboardjs as one of the frontend technologies.

---
**Contributor Resources**
- [Meshery documentation site](https://meshery.layer5.io/docs/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by aditya2499**
**Description**

This PR fixes #1448 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -128,7 +128,9 @@ Meshery uses adapters to provision and interact with different service meshes. F
 _Tip:_ The [Meshery adapter for Istio](https://github.com/layer5io/meshery-istio) is a good reference adapter to use as an example of a Meshery adapter written in Go.
 
 ## <a name="contributing-ui">UI Contribution Flow</a>
-Meshery is written in `Go` (Golang) and leverages Go Modules. UI is built on React and Next.js. To make building and packaging easier a `Makefile` is included in the main repository folder.
+Meshery is written in `Go` (Golang) and leverages Go Modules. UI is built on React, Billboardjs and Next.js. To make building and packaging easier a `Makefile` is included in the main repository folder.
```
Comment (kushthedude) [inline CONTRIBUTING.md:131]:
```suggestion
Meshery is written in `Go` (Golang) and leverages Go Modules. UI is built on React, Billboard.js, and Next.js. To make building and packaging easier a `Makefile` is included in the main repository folder.
```

Comment (aditya2499) [reply CONTRIBUTING.md]: corrected.

**Comment by PluckyPrecious** [comment]
Looks good @aditya2499 🥳

**Review by kushthedude — CHANGES_REQUESTED** [review]
You have used the screenshot of the website rather than the meshery application aligned in the center. Please update the screenshot with the proper image.

**Comment by aditya2499** [comment]
@kushthedude I have changed the image.

**Review by kushthedude — APPROVED** [review]
great work @aditya2499 !

**Pull request merged**